x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java

```diff
@@ -780,6 +786,23 @@ public void testKibanaSystemRole() {
             assertThat(kibanaRole.indices().allowedIndicesMatcher(UpdateSettingsAction.NAME).test(indexAbstraction), is(false));
             assertThat(kibanaRole.indices().allowedIndicesMatcher(RolloverAction.NAME).test(indexAbstraction), is(false));
         });
+
+        // Ensure privileges necessary for ILM policies in APM & Endpoint packages
+        Arrays.asList("metrics-apm.app-*", "metrics-apm.internal-*", "metrics-apm.profiling-*", "logs-apm.error_logs-*", "traces-apm-*")
```

Comment:
These should be conrete index names instead of wildcards, e.g. 
```java
"metrics-apm.app-" + randomAlphaOfLengthBetween(3, 8)
```
instead of
```java
"metrics-apm.app-*"
```